libbpf-tools/biostacks.bpf.c

```diff
@@ -110,4 +97,47 @@ int BPF_PROG(blk_account_io_done, struct request *rq)
 	return 0;
 }
 
+
+SEC("fentry/blk_account_io_start")
+int BPF_PROG(blk_account_io_start, struct request *rq)
+{
+	return trace_start(ctx, rq, false);
```

Comment:
Missing the `true` case. `fentry` progs should have as many as the `kprobe` ones.

Reply:
blk_account_io_{start, done} were renamed to __blk_account_io_{start, done} from v5.16, see commit a184f095ecb4(libbpf-tools: Fix bio tools).
bpf_program__set_attach_target() can be used for that,  `fentry` progs are not always same as `kprobe`

Reply:
I mean `blk_account_io_merge_bio` is missing.